### ISSUE TITLE
Illegal division by zero at /opt/pgbadger/pgbadger line 7822 in incremental mode

### ISSUE THREAD

**Issue by aleszeleny** [darold/pgbadger#434]
Error captured on sample data:
Log line prefix: `'%t %p %c %l %r %d %u %x %v %a %i [%e]:'`
```
2018-09-10 06:30:04 UTC 32538 5b9236d9.7f1a 293    0    [00000]:LOG:  checkpoint starting: time
2018-09-10 06:32:35 UTC 32538 5b9236d9.7f1a 294    0    [00000]:LOG:  checkpoint complete: wrote 1504 buffers (0.3%); 0 WAL file(s) added, 1 removed, 0 recycled; write=150.657 s, sync=0.005 s, total=150.905 s; sync files=27, longest=0.002 s, average=0.000 s; distance=13453 kB, estimate=293637 kB
2018-09-10 07:00:04 UTC 32538 5b9236d9.7f1a 295    0    [00000]:LOG:  checkpoint starting: time
2018-09-10 07:03:23 UTC 32538 5b9236d9.7f1a 296    0    [00000]:LOG:  checkpoint complete: wrote 1991 buffers (0.4%); 0 WAL file(s) added, 1 removed, 0 recycled; write=199.405 s, sync=0.002 s, total=199.445 s; sync files=47, longest=0.001 s, average=0.000 s; distance=24340 kB, estimate=266707 kB
2018-09-10 07:30:04 UTC 32538 5b9236d9.7f1a 297    0    [00000]:LOG:  checkpoint starting: time
2018-09-10 07:32:30 UTC 32538 5b9236d9.7f1a 298    0    [00000]:LOG:  checkpoint complete: wrote 1455 buffers (0.3%); 0 WAL file(s) added, 1 removed, 0 recycled; write=145.625 s, sync=0.002 s, total=145.650 s; sync files=31, longest=0.002 s, average=0.000 s; distance=14667 kB, estimate=241503 kB
2018-09-10 08:00:04 UTC 32538 5b9236d9.7f1a 299    0    [00000]:LOG:  checkpoint starting: time
```

Non incremental parsing:
```
postgres@monitoring-influx-poc:/storage/pgbadger/reports/testcase$ rm -rf outd/; mkdir outd; perl /opt/pgbadger/pgbadger -p '%t %p %c %l %r %d %u %x %v %a %i [%e]:' testcase.log -f stderr -O $PWD/outd
[========================>] Parsed 1275 bytes of 1275 (100.00%), queries: 0, events: 0
LOG: Ok, generating html report...
```
With incremental option:
```
postgres@monitoring-influx-poc:/storage/pgbadger/reports/testcase$ rm -rf outd/; mkdir outd; perl /opt/pgbadger/pgbadger -p '%t %p %c %l %r %d %u %x %v %a %i [%e]:' testcase.log -f stderr -I -O $PWD/outd
[========================>] Parsed 1275 bytes of 1275 (100.00%), queries: 0, events: 0
LOG: Ok, generating HTML daily report into /storage/pgbadger/reports/testcase/outd/2018/09/10/...
Illegal division by zero at /opt/pgbadger/pgbadger line 7822.
```

Used pgbadger commit: `b55d164`, same for `b9d8d38`.
Hope this help.
Ales

**Comment by darold** [comment]
Fixed in commit 5b8c267.

**Comment by aleszeleny** [comment]
Tested, works like a charm. Thanks a lot!
